fastly/computeacls/api_test.go

```diff
@@ -194,6 +194,25 @@ func TestClient_ComputeACL(t *testing.T) {
 			page++
 		}
 	})
+
+	// Lookup a non-existing IP in the test compute ACL
+	fastly.Record(t, "lookup_non_existing_ip", func(c *fastly.Client) {
+		ip, ipNet, err := net.ParseCIDR("73.49.184.42/24")
```

Comment:
According to the API docs, the lookup operation accepts an address, not an address/mask combination. Given that, I don't think there's any need to use `ParseCIDR` here, as the test value should just be `73.49.184.42`.